frontend/nc-games/src/App.js

```diff
@@ -1,23 +1,30 @@
 import './App.css';
-
+import { useState, useEffect } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import * as api from "./api"
+import Welcome from "./components/Welcome";
+import Reviewlist from "./components/Reviewlist";
+//import Categories from "./components/Categories";
 function App() {
+  
+  const [user, setUser] = useState('user');
+  const [reviews, setReviews] = useState([]);
+  const [categories, setCategories] = useState([]);
```

Comment:
you're not using either of these states you have created - as you've recreated them in other components